src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -135,7 +135,7 @@ static bool addTriggeredSubscriptions
   BSONArrayBuilder attrA;
   for (unsigned int ix = 0; ix < cr.contextRegistrationAttributeVector.size(); ++ix)
   {
-    ContextRegistrationAttribute* craP = cr.contextRegistrationAttributeVector.get(ix);
+    ContextRegistrationAttribute* craP = cr.contextRegistrationAttributeVector.operator[](ix);
```

Comment:
This might work, but it's not what we want.
These calls (all of them) should be made like this:

    cr.contextRegistrationAttributeVector[ix];

Otherwise, we don't really gain anything on the change ...